src/Helper/Helper_PDF_List_Table.php

```diff
@@ -228,6 +228,8 @@ public function column_cb( $item ) {
 			 data-fid="<?= $form_id ?>"
 			 src="<?= $this->gform->get_plugin_url() ?>/images/active<?= intval( $is_active ) ?>.svg"
 			 alt="<?= $label ?>"
+			 role="img"
+			 aria-label="<?= $label ?>"
```

Comment:
`role="img"` definitely isn't the right approach here as it's assigned to a single image and not a container with a "group" of images https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Role_Img

. `arial-label` not correct either. Likely need aria-live on a new visible element: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions. Since GF has updated how the toggle looks this should be much easier to achieve.